### ISSUE TITLE
ice6/opensheetmusicdisplay - new name?

### ISSUE THREAD

**Issue by AlbertHart** [ice6/opensheetmusicdisplay#2]
Can we rename the Repository from ice6/opensheetmusicdisplay to something else,
like ice6/**new_osmd_demo**?

That would make it a little less confusing for use for the OSMD folks when we show it to them.

Or perhaps something else,,,

Now that you have shown an interest in my "notation" stuff, I think we should put that in this github project as well. Since I am hoping there are other people who would like to use them.

- We will have a routine which adds bass to leadsheets (treble clefs) based on what we discussed.

- We will have a routine which creates a "voice leading" accompaniment - in either treble or bass clef - which provides chords with voice leading for minimum hand movement. 

- We will have a routine which produces a "comping" score - which has optional lyrics, and shows chords divided into measures properly - with repeats, codas, etc. but no notes.
(I simply load the music xml file with note invisible)

Several of these routines are in this PDF - created by manipulating musicxml and loading into OSMD.

[fields-reharm-osmd.pdf](https://github.com/ice6/opensheetmusicdisplay/files/4653716/fields-reharm-osmd.pdf)

So I think you should change the name of this, full repository to something like ice6music. What does ice6 mean? I see that ice6music.com is available so we could grab that domain if we want to make these routines available.

Let me know what you think. I think it would be fun tio collaborate on a website for music people who want to do things starting with musicxml files.

**Comment by AlbertHart** [comment]
Since the "improvisation" section is also generated by OSMD, it should be easy to add guitar chord symbols to it as well. I use it for both piano and guitar which I want to be able to accompany it without being distracted by actual notes. 

![image](https://user-images.githubusercontent.com/8965352/82387074-4449a080-99f3-11ea-94af-aa236c1973c0.png)

Similarly I don't really want to see guitar chord symbols as I play - but we can make that an option, And we can also show the guitar symbols in the chord section at the bottom.

![image](https://user-images.githubusercontent.com/8965352/82393801-aeb70c80-9a04-11ea-871c-6db5b1c6dc8a.png)

**Comment by ice6** [comment]
about the name,  we can decide this after `Request to OSMD` [https://github.com/ice6/opensheetmusicdisplay/issues/4](https://github.com/ice6/opensheetmusicdisplay/issues/4)

about the domain, maybe `github pages` will work for the routines we planned. it is quite convinent. how do you think?

**Comment by AlbertHart** [comment]
I agree about keeping the current repository naming until after we hear from OSMD about merging our stuff in.

Fur our other applications we could set up a new repository - which had out stuff, but only a static copy of the OSMD routines.

I was thinking about continuing to use GIthub pages, but if we purhase a domain name like ice6music.com, we can register that as the domain naoe for github pages. Then we could have a website like ice6music.com that would only display the github pages - but also allow other code in sub-domains it we wanted - like forum.ice6music.com.

> About custom domains and GitHub Pages→
> GitHub Pages supports using custom domains, or changing the root of your site's URL from the default, like octocat.github.io, to any domain you own.

![image](https://user-images.githubusercontent.com/8965352/82407944-b340ec80-9a27-11ea-8de3-e3f2c949df0c.png)

**Comment by AlbertHart** [comment]
I'd still like to pursue a new name for joint software we might pursue as a continuation after the transpose stuff.

Does ice6 have any particular meaning to you?

**Comment by ice6** [comment]
> Does ice6 have any particular meaning to you?
ice6 has no particular meaning for me. it is short and easy to remember.
actually it is a bit [Chinglish](https://en.wikipedia.org/wiki/Chinglish)

ice is 🧊冰(bing), 冰 is my firstname.
6 sounds 刘(liu) ，刘 is my lastname.

So it has no particular meaning for me :P, just a nickname used in web.

> I'd still like to pursue a new name for joint software we might pursue as a continuation after the transpose stuff.

yeah, we can discuss throught zoom after the transpose stuff :)

**Comment by AlbertHart** [comment]
If you were to express your name in an image, would it look something like this.

Or can you do it better?

![image](https://user-images.githubusercontent.com/8965352/83926846-4a0be980-a748-11ea-8b3a-b207698a9e36.png)

**Comment by ice6** [comment]
@AlbertHart thank you, it is good :)  it depends where do you place it :P

**Comment by AlbertHart** [comment]
I mainly used it in out favicon and our logo.

![image](https://user-images.githubusercontent.com/8965352/83988761-7c147b80-a901-11ea-9258-200be91b7376.png)



![image](https://user-images.githubusercontent.com/8965352/83988737-699a4200-a901-11ea-9c89-f514f2a96b01.png)

But you can make any changes you like.

**Comment by ice6** [comment]
thank you :)
